independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/codegen/CreateProjectHelper.java

```diff
@@ -28,6 +28,8 @@
     public static final String DETECT_JAVA_RUNTIME_VERSION = "<<detect java runtime version>>";
     private static final Pattern JAVA_VERSION_PATTERN = Pattern.compile("(\\d+)(?:\\..*)?");
 
+    private static final int MAX_LTS_SUPPORTED_BY_KOTLIN = 17;
```

Comment:
Can you move this one just below `JAVA_VERSIONS_LTS`? Looks like a detail but it will improve our changes to keep it up to date.

Reply:
sure, done